gcp/opsverse-gke/vars.tfvars

```diff
@@ -1,10 +1,20 @@
-gcp_project_id      = "opsverse-dev-poc"
-gcp_region          = "us-east4"
-cluster_name        = "opsverse-gke-cluster"
+gcp_project_id = "opsverse-dev-poc"
+gcp_region     = "us-east4"
+cluster_name   = "opsverse-gke-cluster"
+# The bucket name should be of the format devopsnow-loki-<customer_name>-<stack_name>
+#If customer name is kylaq , the stack name is decided as alpha then bucket_name = "devopsnow-loki-kylaq-alpha" 
+bucket_name = "devopsnow-loki-<customer_name>-<stack_name>"
+
+
+#The customer_stack should be of the format <customer_name>-<stack_name> 
+#If customer name is kylaq , the stack name is decided as alpha then customer_stack = "kylaq-alpha"
+#Please Note: The stack name used here should be the same one when trying to launch the stack through the admin console.
+customer_stack = "<customer_name>-<stack_name>"
```

Comment:
Why is this being called `customer_stack`? This feels ambiguous for the customers. Can this naming convention be simplified?

Reply:
In our org. we follow two different conventions for service accounts created in GCP and Kubernetes, so to address this i used this customer_stack input that binds the gcp sa to the k8s sa. I tried to keep the variable simple while including some logic, so I used customer_stack as customername-_stackname, If you have any better suggestions i am open to it

Reply:
Done